content/en/docs/setup/install/operator/index.md

```diff
@@ -75,25 +73,210 @@ accessible to `istioctl` by using this command:
 $ istioctl experimental profile list
 {{< /text >}}
 
-Optionally, you can use the `-s` flag with a install package path to see the
-list of configuration profiles available for other Istio versions:
+## Customize Istio settings using the `IstioControlPlane` API
+
+You can change a feature or component setting by using the [`IstioControlPlane` API](/docs/reference/config/istio.operator.v1alpha12.pb/)
+([proto](https://github.com/istio/operator/blob/release-1.3/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto)).
+
+### Identify the feature or component
+
+The API groups Istio control plane components by feature, as shown in the table below:
+
+| Feature | Components |
+|---------|------------|
+`Base` | CRDs
+`Traffic Management` | Pilot
+`Policy` | Policy
+`Telemetry` | Telemetry
+`Security` | Citadel
+`Security` | Node agent
+`Security` | Cert manager
+`Configuration management` | Galley
+`Gateways` | Ingress gateway
+`Gateways` | Egress gateway
+`AutoInjection` | Sidecar injector
+
+In addition to the core Istio components, third-party addon features and components are also available:
+
+| Feature | Components |
+|---------|------------|
+`Telemetry` | Prometheus
+`Telemetry` | Prometheus Operator
+`Telemetry` | Grafana
+`Telemetry` | Kiali
+`Telemetry` | Tracing
+`ThirdParty` | CNI
+
+Features can be enabled or disabled, which enables or disables all of the components that are a part of the feature.
+Namespaces that components are installed into can be set by component, feature, or globally.
+
+### Configure the feature or component settings
+
+After you identify the name of the feature or component from the previous table, you can use the API to set the values
+using the `--set` flag, or create an overlay file and use the `--filename` flag. The `--set` flag
+works well for customizing a few parameters. Overlay files are designed for more extensive customization, or
+tracking configuration changes.
+
+The simplest customization is to turn a feature or component on or off from the configuration profile default.
+
+For example, to disable the telemetry feature in a default configuration installation, use this command:
```

Comment:
remove For Example please :)

Reply:
default configuration -> default profile?  Not sure if configuration is correct.

Reply:
Done.